docs.html

```diff
@@ -0,0 +1,119 @@
+                        </ul>
+                  </div>
+            </div>
+            <aside class="sidebar">
+
+                  <h3>Getting Started</h3>
+                  <div class="sidebar-components">
+                        <ul class="bullet-none list-none">
+                              <li>
+                                    <a href="docs.html">Installation</a>
+                              </li>
+                              <li>
+                                    <a href="alert.html">Alert</a>
+                              </li>
+                              <li>
+                                    <a href="badge.html">Badge</a>
+                              </li>
+                              <li>
+                                    <a href="button.html">Button</a>
+                              </li>
+                              <li>
+                                    <a href="avtar.html">Avtar</a>
+                              </li>
+                              <li>
+                                    <a href="image.html">Image</a>
+                              </li>
+                              <li>
+                                    <a href="form.html">Form</a>
+                              </li>
+                              <li>
+                                    <a href="navigation.html">Navigation</a>
+                              </li>
+                              <li>
+                                    <a href="list.html">Lists</a>
+                              </li>
+                              <li>
+                                    <a href="card.html">Card</a>
+                              </li>
+                              <li>
+                                    <a href="model.html">Model</a>
+                              </li>
+                              <li>
+                                    <a href="rating.html">Rating</a>
+                              </li>
+                              <li>
+                                    <a href="slider.html">Slider</a>
+                              </li>
+                              <li>
+                                    <a href="typography.html">Typography</a>
+                              </li>
+                              <li>
+                                    <a href="toast.html">Toast</a>
+                              </li>
+                        </ul>
+                  </div>
+
+            </aside>
+
+            <div class="main-content">
+                  <h2>Introduction To Awesome UI</h2>
+                  <p>Awesome UI is an open source styling framework, which helps Developers to use best styles and
+                        components, by
+                        using pre-defined classnames. Awesome UI helps you to quickly build your Dream Project, by
+                        focusing
+                        much on Functionality, because we take care of your Designs.</p>
```

Comment:
Indentation could be improved in this way

```
<p> 
    Awesome UI is an open-source styling framework, which helps Developers to use the best styles and
    components, by using pre-defined classnames. Awesome UI helps you to quickly build your Dream Project,
    by focusing much on Functionality, because we take care of your Designs.
</p>
```